spec/models/news_item_spec.rb

```diff
@@ -1,5 +1,59 @@
 require 'rails_helper'
 
 RSpec.describe NewsItem, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.